tests/test_ui_manager.py

```diff
@@ -208,7 +210,7 @@ def test_draw_ui(self, _init_pygame, _display_surface_return_none):
         plat = platform.system().upper()
         if plat == 'WINDOWS':
             comparison_surface = pygame.image.load(
-                os.path.join('tests', 'comparison_images', 'test_ui_manager_draw_ui.png')).convert_alpha()
+                os.path.join('comparison_images', 'test_ui_manager_draw_ui.png')).convert_alpha()
```

Comment:
This change makes this test fail when I run the tests via my standard approach of running the command `pytest tests` from the `pygame_gui` root folder. I assume because it isn't now looking in the `tests/` folder. 

I think this would work only if you ran the tests with the working directory set as the `tests/` folder but I believe the CI auto tests are setup the way I describe above for Travis/GitHub Actions so will likely fail on any windows builds that are on those if I try to merge it. 

The windows and linux branches should definitely be the same for sure.

Reply:
I see, I'll change it back then!